tests/utils/pod.go

```diff
@@ -74,18 +79,46 @@ func newExecutorPodWithPVC(podName, cmd string, pvc *k8sv1.PersistentVolumeClaim
 	}
 }
 
+// Finds the first pod which has the passed in prefix.
+func FindPodByPrefix(clientSet *kubernetes.Clientset, namespace, prefix, labelSelector string) (*k8sv1.Pod, error) {
+	var result *k8sv1.Pod
+	err := wait.PollImmediate(2*time.Second, PodCreateTime, func() (bool, error) {
+		podList, err := clientSet.CoreV1().Pods(namespace).List(metav1.ListOptions{})
```

Comment:
We need to add a condition here that verifies only a single pod is created for src and target.

Reply:
Done